shared_model/backend/protobuf/queries/proto_get_account_asset_transactions.hpp

```diff
@@ -35,4 +35,4 @@ namespace shared_model {
   }  // namespace proto
 }  // namespace shared_model
 
-#endif  // IROHA_GET_ACCOUNT_ASSET_TRANSACTIONS_H
+#endif  // IROHA_GET_ACCOUNT_ASSET_TRANSACTIONS_H
```

Comment:
I suggest to remove changes in the file (no changes other than removing newline)